sky/packages/sky/example/big_switch/lib/main.dart

```diff
@@ -0,0 +1,37 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+library big_switch;
```

Comment:
I would just put this as a standalone file in `sky/packages/sky/example/widgets/big_switch.dart`.  That directory has a bunch of random stand-alone examples.  (Also, you can leave off the `library` directive.)